pkg/sfu/buffer/datastats.go

```diff
@@ -1,4 +1,4 @@
-package rtc
+package buffer
```

Comment:
move these yet again as this is used in `buffer` package and that is at the leaf. If not, get the import order cycle.